releases/release-8.3.0.md

```diff
@@ -215,6 +215,7 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v8.3/quick-start-with-
 | [`tidb_gc_concurrency`](/system-variables.md#tidb_gc_concurrency-new-in-v50) | Modified | Starting from v8.3.0, this variable controls the number of concurrent threads during the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) and [Delete Range](/garbage-collection-overview.md#delete-ranges) steps of the [Garbage Collection (GC)](/garbage-collection-overview.md) process. Before v8.3.0, this variable only controls the number of threads during the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) step. |
 | [`tidb_low_resolution_tso`](/system-variables.md#tidb_low_resolution_tso) | Modified | Adds the GLOBAL scope. |
 | [`tidb_opt_projection_push_down`](/system-variables.md#tidb_opt_projection_push_down-new-in-v610) | Modified | Adds the GLOBAL scope and persists the variable value to the cluster. Changes the default value from `OFF` to `ON` after further tests, which means that the optimizer is allowed to push `Projection` down to the TiKV coprocessor. |
+| [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-new-in-v800)    | Modified | The range of values has been modified to either `0` or `[536870912, 9223372036854775807]`. The minimum value is `536870912` bytes (that is, 512 MiB), which is more reasonable. |
```

Comment:
```suggestion
| [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-new-in-v800)    | Modified | The range of values has been modified to either `0` or `[536870912, 9223372036854775807]`. The minimum value is `536870912` bytes (that is, 512 MiB) to avoid setting a cache size that is too small and causes performance degradation. |
```